771.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @param {string} jewels
+ * @param {string} stones
+ * @return {number}
+ */
+var numJewelsInStones = function (jewels, stones) {
+  let count = 0;
+
+  for (let i = 0; i < jewels.length; i++) {
+    for (let j = 0; j < stones.length; j++) {
+      if (jewels[i] === stones[j]) {
+        count++;
+      }
+    }
+  }
+  return count;
```

Comment:
You can also do this with less code
```suggestion
    const ans = stones.split("").filter(stone => jewels.includes(stone));

    return ans.length;
```